CONTRIBUTING.rst

```diff
@@ -0,0 +1,158 @@
+Contribution
+============
+
+This document is an extension to the generic `Anuket Project Operations and Procedures
+<https://wiki.anuket.io/display/HOME/Anuket+Project+Operations+and+Procedures>
+and describes contribution rules to the Anuket specifications, that are the set of
+documents maintained in this repository.
+
+
+Permission levels
+-----------------
+
+There are different permissions (levels) available for contributing into the Anuket specifications:
+- **Outside Collaborators**: This includes general public.
+  - Outside collaborators can create issues and add comments to issues and Pull Requests.
+- **Read, Triage, Write, and Maintain** Permissions, which are explained in
+  `here <https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/
+  repository-permission-levels-for-an-organization#permission-levels-for-repositories-owned-by-an-organization>,
+  will be assigned as follows:
+  - **Read** will be assigned to those who want to create issues, join the discussions, receive notifications,
+    and provide feedback.
+  - **Triage** will be assigned to active contributors who create issues, pull requests and review them.
+  - **Writes** will be assigned to sub-project Co-Leads to allow them to manage issues and pull request,
+    approve them and be auto requested for review.
+  - **Maintain** will be assigned to sub-project Leads to allow them to merge PRs into the repository.
+
+
+Labels
+------
+
+Every Issue and PR should be tagged with one or more of the following labels to indicate their
+status and for the automation pipelines to apply the correct Anuket processes.
+
+- **Backlog**: The Issue will be addressed in future releases.
+- **Major/Minor**: The Issue or PR concerns a major or minor change.
+- **WIP**: The PR is still under development, no reviews/approvals are expected yet.
+- **Returned**: The issue/PR has been sent back to author (e.g. more information needed,
+  not a high priority, no longer valid, etc). Only WSL or TSL can tag an issue/PR as Returned.
+- **Idle**: This label will be **automatically** applied (by a GitHub work flow or by WSL/TS Leads)
+  to both issues/PRs to indicate that no activity has been observed on the issue/PR for some
+  determined time frame. This triggers an action to close the issue/PR.
+- **Unresolved**: This label will be **automatically** applied to PRs to indicate that no consensus
+  has been reached on the PR. This triggers an action to resolve the PR by the TSC.
+- **Forced**: This label will be **automatically** applied to PRs to indicate that the PR has
+  been merged without following the default process (due to reasons identified below)
+
+
+Issues
+------
+
+- Title needs to be in the following format [SPx Chx] (if applicable).
+  - SPx: The name of the sub-project it is targeting.

```

Comment:
Blank line missing

Reply:
@karinesevilla For the sub-bullets a blank line is not required

Reply:
@karinesevilla Thanks - Fixed